decompressed/base/sbin/random_seed

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+. /rom/usr/sbin/random_seed
```

Comment:
Questo file è nel path sbagliato, random_seed è in /usr/sbin

Reply:
non è meglio includere da /rom ? per avere la certezza che si chiama il file originale?

Reply:
Ho messo il commento su questa riga per riferimento al percorso corretto, il problema non è nella riga, ma nella directory alla quale avete caricato questo file qui su gh

Reply:
ah ok si scusa